webapp/components/wpt-processor.js

```diff
@@ -58,6 +73,10 @@ class WPTProcessor extends LoadingState(PolymerElement) {
       </table>
     </template>
 
+    <template is="dom-if" if="[[resultsLoadFailed]]">
+      <div>No runs found.</div>
```

Comment:
Is this really true if loading *failed*? A bit like https://github.com/web-platform-tests/wpt.fyi/issues/1465.

Reply:
Good point. This also isn't showing up when an empty response is successfully returned. I fixed this by splitting this into two separate error prompts.